addon/components/bs-button-group.js

```diff
@@ -160,7 +160,7 @@ export default Ember.Component.extend(SizeClass, {
      */
     value: undefined,
 
-    _syncValueToActiveButtons: Ember.observer('value','childButtons.@each.value',function(){
+    _syncValueToActiveButtons: Ember.observer('value','childButtons.[].value',function(){
```

Comment:
the @each should stay here since your chaining another property https://github.com/emberjs/ember.js/pull/11994